code/obj/machinery/clonepod.dm

```diff
@@ -679,6 +688,22 @@ TYPEINFO(/obj/machinery/clonepod)
 		src.light.disable()
 		src.UpdateIcon()
 
+	proc/remove_upgrade_modules(mob/user)
+		if (src.is_speedy)
+			src.is_speedy = FALSE
+			src.speed_bonus /= 3
+			src.meat_used_per_tick /= 4
```

Comment:
These, and the `meat_used_per_tick` below, should probably be local file `DEFINE`s at least, so that the multiplier applied when they're installed is the same thing.